src/Abstractions/NexusMods.Abstractions.Loadouts/Extensions/AModFileExtensions.cs

```diff
@@ -14,8 +16,8 @@ public static class AModFileExtensions
     /// Checks whether any items in <see cref="AModFile.Metadata"/> are of type
     /// <typeparamref name="T"/>.
     /// </summary>
-    public static bool HasMetadata<T>(this AModFile file) where T : IMetadata
+    public static bool HasMetadata(this File.Model file, IAttribute attribute)
     {
-        return file.Metadata.Any(m => m is T);
+        return file.Contains(attribute);
```

Comment:
This can be removed. I'll probably end up reworking the Stardew Valley diagnostics anyway, to take advantage of the new system.